ui/hooks/useTransactionInsights.js

```diff
@@ -59,6 +61,15 @@ const useTransactionInsights = ({ txData }) => {
     }
   }, [insightSnapIds, selectedInsightSnapId, setSelectedInsightSnapId]);
 
+  useEffect(() => {
+    return () => {
```

Comment:
This seems less than ideal 🤔 

Not sure yet how to improve it though - do we need to delete the interfaces at all?

Reply:
We could keep the interfaces in state but if a snap creates a lot of interfaces it will bloat the state pretty quickly